libs/components/core/src/lib/modules/numeric/numeric.pipe.spec.ts

```diff
@@ -76,6 +76,22 @@ describe('Numeric pipe', () => {
     }).toThrowError();
   });
 
+  it('should properly handle undefined being passed in', async () => {
+    const fixture = TestBed.createComponent(NumericPipeFixtureComponent);
+    const component = fixture.componentInstance;
+    // NOTE: We had a previous issue with change detection and undefined. This issue only appeared in unit tests when auto detecting changes.
+    fixture.autoDetectChanges();
+    component.value = undefined;
+    fixture.detectChanges();
+    await fixture.whenStable();
+
+    // Get formatted date and remove unwanted special characters.
+    const el = document.querySelector('p') as HTMLParagraphElement;
+    const actual = el.innerHTML.trim().replace(/&nbsp;/g, ' ');
```

Comment:
This confuses the test readability; could you update the fixture template to use a `<span>` instead of a `<p>` tag?

Reply:
Done

Reply:
Actually - I didn't' change this to a `span` but I removed the replacement line.

The special characters are added by the formatter whenever the currency symbol is added after numbers when appropriate. We are removing that `nbsp` in other tests appropriately - it just wasn't needed here but was there due to copy pasta.